hack/run_test.sh

```diff
@@ -162,6 +162,17 @@ OUTPUT_FILE="${ARTIFACT_DIR}/output-${1}.log"
 mkdir -p "${ARTIFACT_DIR}"
 bash "$THIS_DIR/print_title.sh" | tee "${OUTPUT_FILE}"
 
+## CI Dashboad files
+git rev-parse --short HEAD > "${ARTIFACT_DIR}/ci_artifact.git_version"
+function create_dashboard_ocp_version() {
+    connection_dir=$1
+    tail -1 "${connection_dir}/OCP_Version.txt" | tr ' ' '\n' | tail -1 > "${ARTIFACT_DIR}/ocp.version"
+}
+function create_dashboard_op_version() {
```

Comment:
The function names are too similar, hard to notice the difference - error prone. WDYT?